### PR TITLE
Create a sentiments example

### PR DESCRIPTION
**Pull request by technovangelist**
A simple example for sentiments analysis and a writer of lists of 10 tweets

### CODE REVIEW

File: examples/sentiments/Modelfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Modelfile for creating a sentiment analyzer. 
+# Run `ollama create sentiments -f pathtofile` and then `ollama run sentiments` and enter a topic
+
+FROM orca
+TEMPLATE """
```
Comment (jmorganca) [inline examples/sentiments/Modelfile:5]: Do we need to use `TEMPLATE` here? This seems like the kind of thing that could go in `SYSTEM` below.

File: examples/sentiments/Readme.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+### Response:
+```
+
+If we just wanted to have the text:
+
+```Plaintext
+You are a sentiment analyzer. You will receive text and output only one word, either POSITIVE or NEGATIVE or NEUTRAL, depending on the sentiment of the text.
+```
+
+then we could have put this in a SYSTEM block. But we want to provide examples which require updating the full Template. Any Modelfile you create will inherit all the settings from the source model. But in this example, we are overriding the Template.
```
Comment (jmorganca) [inline examples/sentiments/Readme.md:23]: Similar to above, editing TEMPLATE would really require an understanding of what prompt format the model is expecting, but that might not be the case here?

Comment (technovangelist) [reply examples/sentiments/Readme.md:23]: i dont think any of the examples have a template set

File: examples/sentiments/Modelfile (hunk 1 above)
Comment (technovangelist) [reply examples/sentiments/Modelfile:5]: i seem to remember the examples wouldn't work without splitting it out from system. plus this is the only example with template. we document but don't show it

**Review by jmorganca — APPROVED** [review]

**Pull request merged**